demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1018,3 +1022,52 @@ def test_IsValidVersionValidator_fix():
         == "Updated the content item version to -1."
     )
     assert content_item.version == -1
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_number_of_failures, expected_error_message",
+    [
+        pytest.param(
+            [create_integration_object()],
+            0,
+            "",
+            id="valid integration",
+        ),
+        pytest.param(
+            [create_integration_object(paths=["display"], values=["partner"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("partner"),
+            id="invalid integration",
+        ),
+        pytest.param([create_playbook_object()], 0, "", id="valid playbook"),
+        pytest.param(
+            [create_playbook_object(paths=["name"], values=["community"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("community"),
+            id="invalid playbook",
+        ),
+        pytest.param([create_script_object()], 0, "", id="valid script"),
+        pytest.param(
+            [create_script_object(paths=["name"], values=["community"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("community"),
+            id="invalid script",
+        ),
+        pytest.param([create_classifier_object()], 0, "", id="valid classifier"),
+        pytest.param(
+            [create_classifier_object(paths=["name"], values=["partner"])],
+            1,
+            ERROR_MSG_TEMPLATE.format("partner"),
+            id="invalid classifier",
+        ),
+    ],
+)
+def test_IsEntityNameContainExcludedWordValidator(
```

Comment:
Please add doc strings with the cases.

Reply:
I use pytest.param so in the id you can see an explanation for every case

Reply:
Never mind, I added a doc strings with the cases